css/style.css

```diff
@@ -0,0 +1,195 @@
+  height: 48px;
+  width: 100%;
+}
+
+.nav_mobile {
+  color: #fff;
+  display: flex;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  position: absolute;
+  font-size: 18px;
+  font-weight: bold;
+  font-family: sans-serif;
+  left: 30px;
+}
+
+.nav_mobile-btn {
+  position: absolute;
+  font-size: 40px;
+  right: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  position: absolute;
+  width: 375px;
+  height: 1316.15px;
+  top: 19px;
+}
+
+.presentation {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: static;
+  width: 340px;
+  height: 210px;
+}
+
+.my_name {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 46px;
+  line-height: 72px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #7d73fc;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.roll {
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #edecff;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.sup_text {
+  display: flex;
+  flex-direction: column;
+}
+
+.st {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #c8c8d4;
+  width: 327px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.divider_a {
+  position: absolute;
+  top: calc(50% - 265px);
+}
+
+.divider_b {
+  position: absolute;
+  top: calc(50% - 265px + 505px);
+}
+
+.sup_skills {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  position: static;
+  width: 303px;
+  height: 462.15px;
+}
+
+.sup_list {
+  list-style-type: none;
+  width: 100px;
+}
+
+.sup_li {
+  font-size: 15px;
+  line-height: 24px;
+  color: #f5f5ff;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.list_title {
+  font-size: 12px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #9490ff;
+}
+
+.sup_items {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 104px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  align-content: center;
+  text-align: center;
+}
+
+.social_media {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  align-items: center;
+  height: 40px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.work-btn {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-around;
+  width: 77px;
+  height: 40px;
+  align-self: center;
+}
+
+.work-btn a {
+  width: 47px;
+  height: 32px;
+  top: 4px;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+  text-decoration: none;
+}
+
+.top_circles {
+  position: absolute;
+  width: 513px;
+  height: 563px;
+  left: 5px;
+  top: 97px;
+}
+
+.elipse_1 {
+  position: absolute;
+  width: 160px;
+  height: 160px;
+  left: -106px;
+  filter: blur(50px);
+  background-color: #edecff;
+}
+
+.elipse_2 {
+  position: absolute;
+  width: 68px;
+  height: 68px;
+  left: 336px;
+  top: 495px;
+  filter: blur(50px);
+  background-color: #edecff;
+}
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- [ ] I strongly recommend using the fonts that are provided in the Figma design
- [ ] Please make the headline section responsive as much as you can, I would suggest giving 100% to the width so your width is not fixed and can match different mobile screen sizes.
Your Implementation | Figma Design
:---:|:---:
<img width="136" alt="image" src="https://user-images.githubusercontent.com/77190203/149217049-40788d9c-8c9a-4928-b01b-ea050abb4c8f.png"> | <img width="123" alt="image" src="https://user-images.githubusercontent.com/77190203/149216890-b917f5dd-bba3-43b9-b65d-ba6de34b1776.png">

Reply:
- [x] Issue well resolved ✔️